bbtools/38.96/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:xenial
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="BBTools"
+LABEL software.version="38.96"
+LABEL description="A set of tools labeled as \"Bestus Bioinformaticus\""
+LABEL website="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/"
+LABEL license="https://jgi.doe.gov/disclaimer/"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+LABEL maintainer2="Padraic Fanning"
+LABEL maintainer2.email="faninnpm AT miamioh DOT edu"
+
+RUN apt-get update && apt-get install -y build-essential \
+    openjdk-8-jre-headless \
+    wget && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get autoclean
+
+# get miniconda and the artic-ncov19 repo
```

Comment:
```
# get miniconda and the artic-ncov19 repo
```

Should probably be 
```
# get miniconda
```

because I don't see you installing artic anywhere

Reply:
The "artic" portion is probably left over from when @AbigailShockey was tinkering with a previous version of this Dockerfile.

```suggestion
# get miniconda
```